app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+end
```

Comment:
- [ ] According to the ERD diagram, a `post` should have many `likes` and `comments`. Please set up the association between a `post` and its `likes` and `comments`
![image](https://user-images.githubusercontent.com/46839548/235632123-cd7dc7a5-cd25-4402-bee2-ce465e88a518.png)
- [ ] Your `Post` model is missing a method that updates the posts counter for a user as per requirement. Please implement this method 🙏 
- [ ] It is also missing a method that returns the 5 most recent comments for a given post as stated in the project requirements
![image](https://user-images.githubusercontent.com/46839548/235635791-987a1f4b-bcce-404e-b817-053c03e30b55.png)